app/src/main/java/com/github/se/travelpouch/MainActivity.kt

```diff
@@ -20,7 +20,11 @@ import com.github.se.travelpouch.model.documents.DocumentViewModel
 import com.github.se.travelpouch.model.events.EventViewModel
 import com.github.se.travelpouch.ui.authentication.SignInScreen
 import com.github.se.travelpouch.ui.dashboard.AddActivityScreen
+
+import com.github.se.travelpouch.ui.dashboard.EditActivity
+
 import com.github.se.travelpouch.ui.dashboard.CalendarScreen
+
```

Comment:
```suggestion
import com.github.se.travelpouch.ui.dashboard.EditActivity
import com.github.se.travelpouch.ui.dashboard.CalendarScreen
```

This is purely aesthetic, but remove the empty lines pwease :3